src/components/Cards/Card.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from "react";
+import styled from "styled-components";
+
+
+
+export interface CardProps {
+  title: string;
+  img: string;
+  alt?: string;
+  body?: string;
+  icon?: string;
+  price?: number;
+  restaraunt?: string;
+  isDish?: string;
+  isRest?: string;
```

Comment:
Are these intentionally strings?
If you are using as boolean specify boolean

Reply:
Well spotted, I'm not entirely sure how will the json will look like, probably boolean is preferred (thus the prop name).